client-src/elements/chromedash-gate-column.js

```diff
@@ -146,12 +152,14 @@ export class ChromedashGateColumn extends LitElement {
     this.gate = gate;
     const featureId = this.feature.id;
     Promise.all([
+      window.csClient.getFeatureProgress(featureId),
```

Comment:
Is this redundant with the following line?  Oh, subtle, "Progress" vs "Process".  Hmm, I have confused those before. I wonder if one of them could be changed to something more different.   Since "process" is also a verb, maybe there is a similar word that is a noun only.  Just something to think about for now.

Reply:
Yeah, it is a bit of a tongue twister.  I'll make a note to myself to revisit this naming.  Also, once we get the fields to be organized by gate, then these items will become less important compared to the progress through the set of needed gates.  Once we get to that point, I might have more insight into a name that differentiates the two.